sample_attributes.yml

```diff
@@ -4,9 +4,10 @@ registry_cert_path: /etc/docker/certs.d
 registry_name: /etc/docker/certs.d/registry_hostname:port
 registry_ca_file: /etc/docker/certs.d/registry_hostname:port/ca.crt
 container_user: vagrant
-container_capadd: null
+container_capadd: ''
```

Comment:
Should we make this part of the inspec.yml? Then the benchmark would at least run out-of the box? e.g.

```
  - name: container_capadd
    required: true
    description: 'define needed capabilities for containers.'
    default: ''
    type: string
```

Reply:
We could add a more detailed value like `NET_ADMIN,SYS_ADMIN` here

Reply:
yes please :) , will do